studio-docs/source/org/members.mdx

```diff
@@ -57,7 +57,7 @@ When `Contributors` in an organization create a new graph, they are automaticall
 | Manage Explorer settings (URL, etc)                             |     ✓     |      ✓      |  Non-protected variants only      |          |          |
 | Create [Deployed Graphs](/org/graphs/)                          |     ✓     |      ✓      |      ✓      |          |          |
 | Run schema checks                                               |     ✓     |      ✓      |      ✓      |    ✓     |          |
-| See the schemas of implementing services in federated graphs    |     ✓     |      ✓      |      ✓      |    ✓     |          |
+| See the schemas of subgraphs in federated graphs    |     ✓     |      ✓      |      ✓      |    ✓     |          |
```

Comment:
Did we want to keep these markdown tables aligned? (I could understand either way, it just looks aligned right now.)

```suggestion
| See the schemas of subgraphs in federated graphs                |     ✓     |      ✓      |      ✓      |    ✓     |          |
```